src/components/ha-card.ts

```diff
@@ -32,6 +32,10 @@ export class HaCard extends LitElement {
         display: block;
         transition: all 0.3s ease-out;
         position: relative;
+        margin: var(
+         --ha-card-margin, 
+         var(--card-margin, 0px)
+        );
```

Comment:
why the --card-margin? We shouldnt create two new vars?

Should just be?  right
```suggestion
        margin: var(
         --ha-card-margin, 
         0px
        );
```

Reply:
```suggestion
        margin: var(--ha-card-margin, 0px);
```